common/metrics/defs.go

```diff
@@ -743,6 +745,10 @@ const (
 	OutputhostCGAckMgrResetMsgError
 	// OutputhostCGSkippedMessages is the gauge to track skipped messages
 	OutputhostCGSkippedMessages
+	// OutputhostCGAckMgrSeqNotFound is the gauge to track acks whose seq number is not found
+	OutputhostCGAckMgrSeqNotFound
```

Comment:
Since it will be reset now and then Gauge made sense here.. we already have a CreditsReceived Counter to track the credits received here.. this is locally accumulated credits when it cannot be sent to the write pump at the moment..